tests/test_functional.py

```diff
@@ -323,3 +331,10 @@ def test_obal_bump_release_hello():
     assert_obal_success(['bump-release', 'hello'])
 
     assert '2' == subprocess.check_output(['rpmspec', '-q', '--queryformat=%{release}', '--srpm', '--undefine=dist', 'packages/hello/hello.spec'], universal_newlines=True)  # noqa: E501
+
+
+@obal_cli_test(repotype='upstream')
+def test_obal_lint_hello():
+    assert_obal_success(['lint', 'hello'])
+
+    assert_mockbin_command(["rpmlint"])
```

Comment:
Wouldn't the existing command be sufficient?
```python
assert_mockbin_log(['rpmlint packages/hello'])
```

Reply:
Would it? the failure was this (which includes a tmp path)

```
E             - rpmlint /tmp/tmp_40bcjyl/repo/upstream/packages//hello/
E             + rpmlint
```

Reply:
Oh, you should use `{pwd}/packages/hello` instead.